nautobot/docs/release-notes/version-1.0.md

```diff
@@ -184,6 +184,7 @@ The "Related Devices" table has been removed from the detailed Device view.
 ### Changed
 
 - [#423](https://github.com/nautobot/nautobot/pull/423) - Clarified reference to `/config_contexts/` folder in Git user guide
+- [#467](https://github.com/nautobot/nautobot/pull/467) - Fixes issue with jobs not following field_order
```

Comment:
IMO this belongs either in `Added` (if we are regarding this as a feature) or `Fixed` (if it's a bug fix). My vote would be for the former but I'll accept either. :-)

I'd prefer to reserve `Changed` for genuine behavior changes.

Reply:
Should be in Fixed IMO as there is a reference to it in the examples section of Jobs.

Reply:
I didn't look long enough to find fixed. It should go in fixed, the jobs example uses field_order.